udata/harvest/tests/test_base_backend.py

```diff
@@ -281,6 +287,78 @@ def test_harvest_datasets_get_deleted(self):
         for item in job.items:
             assert item.dataset is None
 
+    def test_no_datasets_duplication(self, app):
+        limit = app.config["HARVEST_AUTOARCHIVE_GRACE_DAYS"]
+        last_update = datetime.utcnow() - timedelta(days=limit - 1)
+        reuse_remote_id_uri = "http://example.com/reuse-this-uri"
```

Comment:
`reuse` means something else in the udata context, it may be misinterpreted :stuck_out_tongue: 
`duplicated_remote_id_uri` maybe?

Reply:
yes indeed, confusing, i'll change it, thanks!